src/Components/FooterComponent/Footer.module.scss

```diff
@@ -0,0 +1,71 @@
+@import "../../styles/colors.scss";
+@import "../../styles/typography.scss";
+
+.footer {
+  display: flex;
+  flex: 1;
+  padding: 2rem 0;
+  border-top: 1px solid #eaeaea;
+  justify-content: space-around;
+  background: $black;
+}
+
+.leftSide {
+  text-align: left;
+}
+
+.center {
+  text-align: left;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.footerNav {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.removeMargin {
+  margin: 0 !important;
```

Comment:
Never use !important. It is a bad practice.
You will have to find an other way to override margins. 
Because we use class based components and modular scss you won't need !important to override anything. Class strategy will always have a priority.

Reply:
Noted